_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Bitcoin Optech Newsletter #109'
+permalink: /en/newsletters/2020/08/05/
+name: 2020-08-05-newsletter
+slug: 2020-08-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains our regular sections with recently
+transcribed talks and conversations, releases and release candidates,
+and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant news about Bitcoin infrastructure development this
+week.*
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+FIXME:michaelfolkson
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [C-Lightning 0.9.0][C-Lightning 0.9.0] is the newest major version of
+  C-Lightning.  It adds support for the updated `pay` command and new
+  `keysend` RPC described in [Newsletter #107][news107 notable].  It
```

Comment:
the "news107 notable" link is a fairly large section; perhaps specific links directly to the PRs, e.g. `/en/newsletters/2020/07/22/#c-lightning-3792` for "new `keysend` RPC", etc.

alternatively, s/described/, which are both described/ to make clear that the general link refers to both the pay command and the keysend RPC.